crates/rpc-types-eth/src/block.rs

```diff
@@ -70,9 +68,10 @@ impl<T> Block<T> {
 
 /// Block header representation.
 #[cfg_attr(any(test, feature = "arbitrary"), derive(arbitrary::Arbitrary))]
-#[derive(Clone, Debug, Default, PartialEq, Eq, Hash)]
+#[derive(Clone, Debug, Default, PartialEq, Eq, Hash, RlpDecodable, RlpEncodable)]
```

Comment:
no rlp for rpc types

Reply:
Cool, got it. Will change. 

What would be the best way to calculate the rlp encoded header for the `rlp` field? 

Ty 🙏

Reply:
Good point, my mistake. Fixed.